### ISSUE TITLE
LANG vs LANGUAGE

### ISSUE THREAD

**Issue by PF4Public** [caffeine-ng/caffeine-ng#6]
Like I said, on my laptop it is `LANGUAGE` that changes the language, not `LANG`.
Have you tried this on your machine? If `LANG` works for you, we can keep both.

**Comment by WhyNotHugo** [comment]
`LANG=ru_RU.utf8 caffeine` works for me. Can you check if that works for you as well?

**Comment by PF4Public** [comment]
**LANG=ru_RU.utf8 caffeine**
![screen10](https://user-images.githubusercontent.com/10319700/40323033-e0a230e6-5d3c-11e8-8281-d100329b6774.png)

**LANGUAGE=ru_RU.utf8 caffeine**
![screen11](https://user-images.githubusercontent.com/10319700/40323040-e3f64656-5d3c-11e8-82f8-b41d7c56de83.png)

**Comment by WhyNotHugo** [comment]
Are either `LANG` or `LANGUAGE` define in your shell by default? Eg, `echo $LANG; echo $LANGUAGE`.

[This](https://superuser.com/questions/392439/lang-and-language-environment-variable-in-debian-based-systems) seems relevant.

**Comment by PF4Public** [comment]
Both are defined. 
I didn't get, how that question could be helpful, please elaborate on that.

**Comment by WhyNotHugo** [comment]
`LANGUAGE` overrides `LANG`. If you've set the former, that's why the latter won't work for you. However, this should apply to just about all your applications; are other apps showing in the language you expect with your current env?

**Comment by PF4Public** [comment]
>If you've set the former
Nope, I didn't

**Comment by PF4Public** [comment]
>If you've set the former

Nope, I didn't

**Comment by WhyNotHugo** [comment]
You said in your previous comment that both variables were set.

**Comment by PF4Public** [comment]
>Both are defined.

and

>Nope, I didn't

Do not contradict, are they?

**Comment by WhyNotHugo** [comment]
Lol. Technically they don't. Let me rephrase:

> `LANGUAGE` overrides `LANG`. If you, someone else, or something else set the former, that's why the latter won't work for you. However, this should apply to just about all your applications; are other apps showing in the language you expect with your current env?

**Comment by PF4Public** [comment]
Still I cannot quite understand what you mean.
I have set my default locale to be en_GB and it is so for every application, for the whole system. Why should it be different for any single application?

**Comment by WhyNotHugo** [comment]
> I have set my default locale to be en_GB and it is so for every application, for the whole system.

What are `LANG` and `LANGUAGE` set to? What language is caffeine picking up for you?

> Why should it be different for any single application?

It's shouldn't. Caffeine uses GTK, and it's GTK that does the localization. It's not very clear to me why it's not working for you as expected [yet].

**Comment by PF4Public** [comment]
>What are LANG and LANGUAGE set to? What language is caffeine picking up for you?

LANG=en_GB.utf8
LANGUAGE=en_GB:en

By default caffeine has english UI

>It's not very clear to me why it's not working for you as expected [yet].

Is it working not as expected? Lets find out then :)

**Comment by WhyNotHugo** [comment]
This project has [moved to codeberg](https://github.com/caffeine-ng/caffeine-ng/issues/117).

Follow-up for this issue is at https://codeberg.org/WhyNotHugo/caffeine-ng/issues/6
